ios/brave-ios/Sources/BraveWallet/Crypto/Accounts/Details/AccountDetailsView.swift

```diff
@@ -40,11 +40,13 @@ struct AccountDetailsView: View {
   var body: some View {
     NavigationView {
       List {
-        Section {
-          AccountDetailsHeaderView(account: account)
-            .frame(maxWidth: .infinity)
-            .listRowInsets(.zero)
-            .listRowBackground(Color(.braveGroupedBackground))
+        if !account.address.isEmpty {
```

Comment:
Good catch, opened [a separate issue](https://github.com/brave/brave-browser/issues/37029) to remove / cleanup this view now that Deposit support added to iOS (export opens export private key modals directly instead of this 'Edit' modal) as that change doesn't need added to this Add Bitcoin Account(s) PR. Could we just add a comment here / somewhere in this file: `TODO: Cleanup with brave-browser#37029` 🙂.